src/proj.cpp

```diff
@@ -196,20 +211,22 @@ bool CPL_have_datum_files(SEXP foo) {
 }
 
 // [[Rcpp::export]]
-Rcpp::NumericMatrix CPL_proj_direct(Rcpp::CharacterVector from_to, Rcpp::NumericMatrix pts) {
+Rcpp::NumericMatrix CPL_proj_direct(Rcpp::CharacterVector from_to, Rcpp::NumericMatrix pts, Rcpp::IntegerVector keep) {
 
 	using namespace Rcpp;
 
 	if (from_to.size() != 2)
 		stop("from_to should be size 2 character vector"); // #nocov
 	if (pts.ncol() != 2)
 		stop("pts should be 2-column numeric vector"); // #nocov
+        if (na.size() != 1)
```

Comment:
na is not known: do you mean keep?

Reply:
Sorry, yes that should be 'keep' where I have 'na'.  This is a remnant; at first I named the argument differently, but forgot to change it.  I am not sure of the best argument name, actually.  So, line 222 needs `s/na/keep` and so does line 223.

I'm not too sure how to make a PR upon a PR, and since this is just two edit commands, I hope stating this is sufficient.

I see that you've pushed recently and travisCI is checking now.

Also, and sorry for this, but I polluted your files with one ending in `~`.  (That's because I use the vim editor, and I forgot to clean the file before doing my `git add`.  I normally have that listed in my `.gitignore` file.)  It was rude of me not to check what I was adding to the git repo.